theano/sandbox/cuda/elemwise.py

```diff
@@ -66,7 +66,7 @@ class NaiveAlgo(object):
     def cache_version(self):
         ver = self.scalar_op.c_code_cache_version()
         if ver:
-            return (19, self.verbose, self.sync, ver)
+            return (20, self.verbose, self.sync, ver)
```

Comment:
I think it is better practice, in case I made mistake in my comment that make compilation crash.